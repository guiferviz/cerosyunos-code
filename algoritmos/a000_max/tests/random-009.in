31
95
1108003467 703699990 869573981 1981938225 668542285 1746231591 63483428 1474837102 791979952 65940152 2028393890 1767402533 1282430059 221836174 1919411763 210869261 461245688 507517785 2065704736 1691270540 412563186 800507898 983350559 2088264464 1367258641 456742840 1658465701 1162467781 1574905939 1094939942 811662443 2136293171 1156666062 1585886169 1200074720 1562503299 253140868 1851296190 1370724577 1782264721 1666425530 1676199910 1830756005 293970764 1227669491 564954569 434774234 795489484 444909230 256507012 1896836974 834307624 244780495 1175877329 599578580 597426142 978486883 615451787 813722391 657384294 2062077257 66705746 1276719991 1500217635 1822431180 1740345592 74036219 861261997 447009652 70328434 955322404 1002239437 2058026455 631135947 1379078089 1882073882 2071169459 1416657556 1641388199 401318040 658190944 1232483167 1784926145 1674075924 296335590 1277788062 169152796 696686095 1109206102 774496400 880494732 730301464 261251051 1811946874 1883886687
95
1073605574 1305465110 920744891 1424611477 638147804 1188278889 578384484 697909077 1591603618 1684143771 1725404835 263485223 1506268502 1590891361 286074378 754933595 2049793079 1809570580 96966642 1488872444 1188196952 1914195480 83776428 131850169 1159115905 1143004581 653457327 1621877600 217126219 40030701 1542635664 2072936011 1291505272 1654804093 328466472 320256387 1356322169 1720637120 127897169 2038678862 171182731 955094538 343975394 527316900 1962509189 831057136 1152318747 1902689168 145123700 850087910 1262549933 1031111979 251515954 1935497764 859377275 79068848 1271851556 1652346966 523552936 1523892790 761238284 1202131371 1776771280 1868729267 242530486 676505499 1025560737 1252233051 2014893594 280192541 1652668245 1750012637 15468051 513488300 970071643 1380572833 1919618140 1162364241 427683130 39131948 1486987157 782368851 787774407 511830695 1384462481 1956510448 973874498 1280779623 1450506743 2016442858 1115228891 1793657045 1633980625 1282523952 905557811
95
1160785177 762620139 572837997 445860564 1937843296 1889528432 1303375117 1814773256 321510021 1298264121 794447135 1673813439 1702977746 1633738758 1684203473 790693407 1905394083 1950408614 1467991868 493323575 1280584489 1731884837 1351004353 95751667 1451346330 1789009498 889527529 285865465 508212136 1861603494 935414346 1305817764 1375428571 344542628 342075889 1474221332 262003902 1866131660 565647947 1371258 2123317720 1324588449 1922859402 1654140832 1812324481 945089139 1441384467 1498645357 1284286537 1855535838 1315952787 1400929290 354162569 1801733880 774188881 1430607068 144605373 2087471847 1850681728 989432272 2007339353 568879914 1506292996 1157934431 1599249855 1604389700 165838966 780060137 1414928338 2064537107 168507367 2142885048 131934528 487193244 1849824453 278395389 1904380757 1781805472 1977266105 1014492176 596330892 1692070992 1036263925 2103789328 1808625399 2090028788 1714854657 61111532 820929294 730091697 855389000 592213017 1421818066 251657679 209422067
95
1411495937 1143828894 816993901 1302381607 488561639 1660889173 929527993 193228140 221875081 613981872 268089981 146160047 533350394 761351375 712713186 1475536981 2034211347 1195523819 1520872761 532825303 1282681282 1631490218 2136886033 622300018 392394839 1502411968 941180497 497913448 953065494 407103420 1805891131 2005261263 1021472251 1563681147 507924218 1330428867 1416807967 1694181843 405611332 1167936538 94792679 1743948637 1424512884 928729958 1506503992 231094435 1801920272 1644465135 1074914195 572472671 1517400273 442046672 2093963967 1552093336 138179254 1328870549 135175467 2124824306 1794967216 672072563 784392831 1821335444 1889297019 642306813 738437005 862132961 1998054159 1792802975 204840063 151429005 1437104446 831317978 1525374645 1476981298 1559283018 547140947 1060813373 1096787308 60943675 10502390 1655601548 1363267291 1687699176 1163704134 119584000 1627639573 686381229 847673395 1424667657 854177228 2017403398 1803934866 613788888 1370787837 335910483
95
42787922 452686975 361134735 1849633164 922788547 1021149039 418093602 1439699365 106976684 722402889 112260832 1400921392 711408220 1502412301 509523526 440878899 251845836 266226707 669302898 60263603 2021456540 1706100867 379852426 1063056345 1453951379 9639084 1018374768 1333709391 417843646 1746078352 1095628362 784132017 1159454685 1936359021 52619063 889905920 1330698450 1922567877 1121110374 357260764 821628639 847716128 2128691439 61750040 2051463546 127927127 178622233 84896578 139945965 669373864 1108165650 382505887 1546302213 214706752 1991041913 1097462287 1779498598 1495756549 1375937359 642625272 362074687 1810311726 1280512618 250991174 1599930873 2047716465 276550018 945543595 526219239 920559294 867002126 592129533 454067451 1327175332 127234761 1913846664 1990932097 114071844 1022313328 1239571649 131470427 261258594 1884120269 1824995784 437825854 167468631 1899325948 1729959082 691041136 1311671526 998296408 253644767 1890198736 707769515 337445834
95
2016623281 542196884 1256122322 1872815057 1750541829 1545410107 360885451 348868664 1729496950 1248373914 329617481 811349667 1450599698 1379274569 667102376 697686236 1606480577 149421577 370313594 2097385186 1915989110 1280206945 1713658803 44654976 1187311188 462796646 1320414350 177009843 367754630 784758521 1215149910 1057105505 2037041399 2042331799 604547931 1452511217 1893903969 572424846 639341038 436795215 1808765655 55389204 1680964360 1880201698 1905000988 312790502 118075694 510361885 574845461 525314638 1009081453 293116025 1045249772 63248213 547696490 967668638 176092421 1718680504 650158039 922039619 1581094774 1000683774 2136026421 1877595366 370544056 1454179803 310346875 1494279177 658792251 1877702751 1855281859 1415174523 1813309590 1794244488 894544894 2008890814 1001861417 1337058888 1902823166 799506062 382958189 1025179935 1590024710 7067448 597983315 296813665 46423438 361957172 437040753 1226780920 1903303132 710775934 744098969 289678249 936421120
95
635987156 1601022567 1797233606 100509299 445161642 576493147 2024942330 1443622037 315560653 1352288437 613419046 1926710600 1393734693 687688845 282214167 1035620593 1294589499 821183412 1031568816 700888813 886878058 1708358642 1974883389 1033920016 265192741 893287722 1400749576 603438744 167000868 209709151 169038496 71292364 534451929 1975382685 2014515279 485331877 277390450 47323700 606387138 164879556 1730955322 1366264897 1824716483 1313391924 1251128514 1118678244 171702144 1291778079 1128938333 1442421354 687178662 869502392 2142279313 1502204637 40013970 778215678 1471380378 824053609 1443403675 1194815505 357799700 1383296872 1594972121 269245506 514195514 221948594 589998889 2039022212 1500411169 375324757 194602144 987447733 435797421 737130283 828094916 812657550 831928626 1275818084 1890894599 1404626821 157799513 1463489747 115752119 1782814680 151337112 855660204 1119001265 101392081 1219710929 807017367 1482442847 1845046082 1317093280 1070800033 249547594
95
1961443489 556111487 953859993 553154564 1082531791 1449073676 644627453 432964014 706598766 606916713 1591132687 394237385 292102255 795805386 917840595 983118849 437156995 647151764 2145494462 911393094 1263410274 532884543 1681652659 2080841584 1886913878 1313080648 982563232 1288128605 627526537 2016546016 385924918 897928708 350399280 1997892437 1854896799 1717160910 1308928130 1769608738 2036477431 1381651972 2052773952 1342540432 1283290093 1450081248 242076578 256812772 686847722 1050498769 82449114 1418316882 1933508123 992943734 2041557053 752491253 597782268 880337928 870362944 325476468 626897935 1109073884 353059546 1556343114 395786897 638148120 903924710 177747143 1479919347 266311522 1029279584 1618077089 1237033665 877323729 753088340 1188232987 1303523157 214485962 21870864 1915924335 847467205 476123355 1983519150 1579011500 521127281 1810937822 1401540021 1768296156 2049171485 499031649 395207207 37944482 1767680001 1296675010 1077673360 150854840 433759762
95
645487049 1035216121 560106452 2142773974 379287856 1541979620 1191553756 1726012189 673295453 2110267281 1783319973 1487662848 1449323873 1960490043 667147538 1638877815 148058193 362019948 2086691464 974961882 262899063 1186485839 1802740920 370025098 170924214 760214269 1183282182 1235011385 1170519790 207427200 312386793 1147755917 1128869191 839155471 373927035 670561688 1738963806 40633997 37668410 1566870251 935235882 486361240 322095370 887820579 1477338807 925131688 90027252 336009749 891352790 1001424932 1459135815 1070236388 639293717 860673166 1499417275 745279543 2052305454 578908449 1507624561 1429574118 1637659657 1011645062 1664887449 1590674402 2144726361 1759138086 1488313557 2102509642 939678619 654489666 332663267 1222283613 1290683361 741867829 109128775 1355427288 333414118 1551715101 536390370 1950583626 1463952422 1151424807 247513948 1979941348 304256774 1725216686 1455573014 411458916 802900441 702625842 1158338863 1348953094 1532067810 1250408471 960507647
95
702376219 1937202918 1148460035 111037578 912750744 430414551 1783209659 2045233135 237959998 1653688505 1988100616 2112566012 1841218545 1260755140 186442658 1167982702 1360665323 1594362065 244456946 29654107 139865670 2044477644 1491312054 787439281 1523563677 1958760166 1855918752 866238388 2060032645 40547790 976223658 2133757847 19874163 1899499019 1986418098 278498636 1164855647 423143059 1334705387 887144231 1217921605 801900752 1085486734 727864129 268358283 1447356223 442316398 1502534401 1003037088 1076608751 1519111420 876682386 868186185 55045761 1284428823 120806715 1677251411 383660286 1873689741 2041331475 1125210945 998892787 46411124 1516019448 586545069 119711375 1951943839 661839817 680495179 26323617 748638704 378161818 1603049756 59250913 1852696272 2001608398 1969642790 1149910155 2041894835 875715621 1542611365 1141517440 952390263 1065725863 1884406828 1524010247 1857761778 872670175 117679601 1784903918 1098248194 2016675926 252363444 1675377697 1144431960
95
1237828985 720557096 838835125 77159201 752608141 1504772897 1541208792 456388880 1689516739 1682319185 1841196745 1817371417 1060492724 1484061003 1375567804 561926600 1004610936 961469019 1871908432 155507651 954667879 1295586582 1507728324 1543773773 1589220284 1712613608 1512140371 1709267766 946118115 1733890503 2143709966 1330035418 195532690 1468895237 804174216 1630265539 229035964 385045321 1248410458 1191029718 140686382 637655925 1915139962 1836563740 1490927901 1064848763 1469329366 143164994 827741336 354359386 1234850692 109742496 1458959038 1473297241 1832017628 262727271 603236239 1603850638 371419318 915294844 1368369818 1833644052 1616929791 128501183 19902286 76779792 1207129870 1477681647 1084619981 1884724754 988992834 1545130777 1092324669 635020132 969473335 1366166449 1549408171 606642173 1826537106 598360801 749567225 1301978739 1558978868 945599622 148762178 439997161 1144637637 468135725 1522455692 1522619573 480639887 1016383153 1532551168 1444746081 1233141214
95
935151850 1207455575 1226973196 2121898685 142109091 319790656 324907636 1852942258 14223369 471493431 175647399 691771393 532343257 1997995969 1581966087 563817080 876002581 1808870805 985389894 1667911881 1344004572 1962764015 1239107968 500024470 835106151 1718306025 1341198672 1892676734 116945950 768300594 1365238879 801893035 1980624526 531367409 1938083987 804931368 731985890 481040610 1060989827 621375046 1594475895 57327817 1348144986 316695609 2033940368 1664619357 1226795017 805121269 1802720286 1337950103 1815963822 648280133 1720884659 222214534 1823012335 1993302805 623559911 762329425 901649549 603928849 1498453678 1039145300 1751570551 326456991 184107496 1642799743 963700653 1985493914 1308817914 1289215339 190559240 1808338145 469234883 584619991 5077538 2098982124 1903522594 446794598 465304736 1960132119 1718210352 412789544 551341379 1450294592 115967237 1702549304 1799530253 556746860 241174357 860754462 1819503443 1553256144 1988481565 1165254358 1654816673
95
727943404 668090515 1538757577 1977294143 902828237 1181345685 141564108 1660551008 2039629514 967010999 1475170476 901922397 1065882481 418514836 1976475318 1640419975 538643564 760257308 1049407598 213179380 1234897429 1728502439 1136560362 2009024753 981889689 207233699 1186853131 973829375 1611766229 2108070539 1860057168 1120383367 1232942655 1558279732 1938806414 1850330557 1969667913 1803587829 1066063922 99129688 1758826527 199980462 46840199 303716659 859419631 550833038 1407053023 846610319 1947823997 550479843 1659361658 552810591 2014780077 1093982548 495744005 693376630 258765243 906843779 1025085022 2098054945 1780941145 2113450068 1823378633 1704167913 191427772 1634596485 1100906794 25737194 734123887 1306073470 1772047943 940609066 1836852642 220811603 1156526680 1014825257 2059344950 1604832838 111265620 694113460 512979399 617067721 604250002 782479416 724865301 1038046039 86360011 606449133 1432252958 220394768 1570570390 497699563 281981437 482632923 178823865
95
1334767783 111939713 1678141733 1026406140 641628476 1798427933 300788778 592167146 294322327 1637979082 364709227 1174129857 1653219659 601892547 567813429 1278282208 1751527223 516009621 1021633643 1422524249 346478279 892059340 739636491 290038945 1091034470 1539650197 2003252426 573861273 1655177754 226811099 1463303662 44054888 1972892503 1806332987 48794508 1884192466 1135776412 148903738 175028972 49412686 1876650638 1250212198 1925099159 1174868157 40041559 1918199399 448686274 1203715929 2008011280 1881300665 818429254 773468860 742493364 1970810558 1894496437 217828431 172102852 796375068 434551057 769735663 130047422 498281560 1520441054 1467666399 2004950474 1658748661 356897844 1736993406 1187693994 1156349281 820758195 521461773 1829576313 1880200782 367075330 1054907828 38588525 2110940202 2060446854 99172507 867823066 2070093420 1749883682 1686976960 1269509460 6030052 1005398207 555283972 163943794 791070783 1639224686 414803961 830152148 462454703 1242147020
95
1592816690 1406959397 220402749 1719693968 597532935 1894484305 1961662596 2097545120 1462149803 1688856535 1641120925 162851899 1668367698 195547400 657688178 32617612 1679600314 180512445 1306487077 2086893547 539996011 241506922 988674612 1991455850 1592953752 2140331603 346405943 1666594899 1565882603 330551256 1514476946 601119866 2040140452 1265670082 938202745 971317519 1065110511 2072018533 1315887515 504855307 357971275 665453284 1461201564 130642965 1059414375 807089935 283507996 1724655427 2087448623 640232170 1120842946 1931486876 1840677606 1677198365 1833829374 1896962707 701660240 1920494277 496694002 27016229 1403905517 1658692606 1465250133 1854833422 1683432922 139801846 464756807 2125789745 1631197700 1530681097 440333373 2045351427 1700506917 2024236485 322487507 174396144 719888534 1990692796 346823199 1795601151 1450302575 911770260 1057116220 722568071 1177937256 93354889 1934678872 1983013017 1628965089 1233523150 1274404279 1206299995 1341182950 94185052 2060837376
95
431465631 939602086 380057285 2090361797 637302344 154406669 194813768 1456811212 1992491234 1885398369 2109824701 1557802604 1673119625 12039745 1816682164 1258143113 2035045652 989468315 1727010339 417569429 1554130202 1778094167 263108930 2141523207 71598310 1589141225 18029768 895791784 2016975875 784635847 1515885356 425261739 749727329 464073798 372455546 193602704 275340837 1859706651 1841831027 1710501271 1208318518 1927839072 95574847 1507240420 1914674542 190332631 729725356 1721077022 7824108 2113736138 1872130847 2079105306 646582755 1176255774 1464334813 655365295 1516863647 270774253 1439763773 2121940079 368359281 92251016 429469361 1095730474 2064830592 1840917454 624148738 1899891170 924855862 1597711434 461340347 2003638366 1619640897 803883604 1418166673 379871694 478223679 857118748 1131049586 1505613794 1619675881 555814272 1835273120 2064498343 354244308 469745340 510298426 334881441 1144915934 2076743514 1493920699 1567951714 1767032015 73518027 1403456583
95
138542328 569492091 1046897733 1018716190 1206447977 1413681943 1614029910 1042352657 956629217 483661196 2141460332 624981288 229092280 446907432 112004320 1754812209 1589643577 192555877 1888225341 1045296302 2039072581 1775419168 129228711 1559166824 407198446 640331928 1818561766 956846225 864647253 1701283709 1628051287 1962861746 831284112 927622215 94585821 712060388 679257395 233671002 2073214670 538366913 70660342 1019951234 1209537527 319275368 1892280197 294629790 1816991690 654248442 1385732904 757407943 295937220 969590699 1527311448 1223450645 950664223 138486816 1856799101 974752726 754140930 1896346046 318545244 1022064818 825025053 34336457 1782190461 1005005449 1194814855 306597545 1186302430 1766077276 995064046 394344388 342384069 698910831 556508054 1215070636 1937658078 1175809262 1419259447 1964392416 158313919 415374360 703703832 539391187 1282343170 962515157 526498030 1841142591 382653632 1988051501 2116799070 1029887226 137286259 368177531 11376089
95
157378242 1127735327 1077947377 176742805 1316553829 524390916 914829568 1537341711 1997665672 1019229421 810461479 1001340039 1333461486 1121407491 562768537 1981496213 449448574 19882492 1762418357 182728718 260870204 1985225734 323434155 987518281 57841375 1639612677 1839438741 790490277 1800505386 1108354975 1666379373 1163991803 237444172 1723332980 558786220 1246623478 953488307 745396490 945862859 2063588090 1183008313 1076803140 1014423772 191776874 1375360387 365189212 653562731 1497512378 5018347 2059538370 189715856 1234713825 424057907 1543576051 2032702413 1235013389 875970379 1035275609 912366855 876150342 1076947970 1036871880 888507008 916659160 1034361964 1734453076 862888470 1609069223 1649417021 1873960202 1934659995 1435118408 909967658 2037640216 747559159 2046412814 183045285 1079228124 1272885180 246035632 2047740967 1353451881 1256403934 1139896165 567317349 1131573120 143753887 343435777 1881101876 562980044 1083322593 281989455 855366111 1728148041 2074117296
95
543716021 719473887 1052956803 724164800 2057036407 1037481817 1843888510 665953729 1424029395 1799755351 1651499720 1168542253 865089596 2106580515 1174746744 1506960052 403153267 311417158 2004040955 121807677 889783476 306517348 1334575290 185246698 590890360 1986282041 1424082033 1404250453 309739853 731401400 254058313 1619437000 218137243 1959255178 606478173 544055618 1222250579 694020810 1501405874 1152178388 1153975723 1995831815 2013638116 773120949 135737385 1137400759 228526336 1550789067 900055393 786093540 1471955109 425737866 1188492339 136514498 1459480992 1893117210 1557070789 1300089178 1873264799 210750092 547117497 1160844485 527559661 295909091 489108267 361181910 1914080368 605040969 513113866 185690582 747168277 1821831999 1931045444 946655951 1857732507 1068620411 801783300 1217030127 1734485900 820863929 2048942252 1915702042 1898339888 56971711 261522051 1557478987 787801076 1079900793 1273562820 2096816548 2101805158 1663708999 88724101 691191682 120955213
95
2095167210 117077813 461776847 1239225017 2013502261 1250875324 681991267 1037106350 1716559175 740745897 1970574324 503039319 204705688 416310646 1955366092 1099167250 1169332123 1676183815 1439338419 553019224 374955384 919874166 1045857076 1571502532 127863259 1198821888 1839650430 910628402 1927652952 1466154953 1836181236 1809849022 233252625 923648747 1294114949 378511765 395907149 144578301 112037062 612381431 1652356716 456442800 333282598 772318827 1098266978 2024326698 307391392 71704564 1181794796 2131745872 841736231 1327713751 2053336547 1296296932 1649031550 667638071 2138510445 1235626724 823776890 942277535 640866436 242458026 860095079 401387865 1148928791 409516683 47553154 1227077837 1902024919 762785068 1427932556 1841616106 1301125971 469968096 522450715 1849464945 1540732930 1234516752 1847222075 2131848317 361560553 1236119454 910967890 502628658 1216510508 1213526647 896508704 1144999390 977785334 961982207 1853714063 336716721 1484711042 1884024315 2006087759
95
1747928430 1810083368 1328755863 961553351 351238558 789566081 578307803 172369090 487032941 1166272434 630763264 1427053093 284789291 827592722 1903955304 1311068602 433337984 1403252737 100236652 1574915067 298283873 376906608 450778107 2071137287 1985607736 128097789 813675569 929934702 420395578 1137741601 1908605759 1379677004 1129245406 311402480 2135085722 76217471 1485770479 779826981 1007734967 649870066 1784976581 992383865 1709931088 752841325 2136904624 664687472 1853076037 712391969 879236441 757630564 1407256225 1051328534 1151661319 2138833021 1668443986 341343463 1908876497 1135054474 2003897531 1526056952 1269701935 398693244 1011681116 1835680057 937198143 1619620869 1811496775 1932534861 786691346 1361541268 180394189 1200275748 1800181080 1305314428 1584751417 1852428131 533848845 1706068383 1961732262 219656695 1733444945 1800888696 1450400259 1181058908 1741325003 359585596 26535598 636364064 1769102640 1849890542 1501559693 2058306620 496404232 1682890037 886259700
95
968760038 128785175 1432269764 1442931141 179514190 1132526957 1304749139 1214837553 2062011675 1019801565 863990204 913247299 413899581 289117186 1424604826 100703286 1356917297 77115425 1744240007 420384445 1413061243 1867902201 344144008 1303863706 1309280944 725620442 1653891447 481598139 1068908834 1549094969 1629917249 18740636 1006134740 1751959746 1376152497 355254869 1217002268 310196920 614422027 1366179513 417036048 1871500774 313687696 836240348 1974503096 2011343440 672793181 1965505923 1288387157 198252513 2105399970 1713284542 1312760888 970865912 939060095 1005304466 1027985481 880928599 1001573523 267765077 849793459 481558409 1796861564 632606355 1326591918 1636480098 844402237 270262882 1540306430 825896205 1334980134 2087871129 1091195692 317287776 1584431889 1115939968 7301223 1147955952 1582443145 251291916 1777796508 1898703210 480273874 581393991 1579296500 989688063 1618013154 1821788963 382872411 221852117 1695070545 1935085910 1641374068 1390961992 2029409148
95
880137499 1569977357 1052870158 1039131048 1104177362 1435805151 1122813186 1558530505 1348754633 1696634983 391756498 494474339 1914683259 2121378875 679122110 814492634 345033431 850745103 705012889 1684140991 1739597337 684336363 2143570519 1981515535 736593736 435674722 1591290838 1707458337 1335634236 77156693 618492373 533309412 1123063404 2006596295 1015671216 171303680 1856064410 1182253160 909475714 138448349 2129202885 1010641859 108197487 1967904441 505752369 1029809888 1841442073 1404817079 1995671969 1384634802 1529366084 102441214 1817144086 553033892 221836044 333865181 1822016813 2144009516 1035136160 999924186 629812392 101729255 592931973 30081874 1028151786 123280890 602117589 701130045 31408717 1153681982 1151865070 503267588 2048104187 1029812768 589561658 1579700770 2122456959 968781202 55149097 976659534 1400388911 1842251401 47825505 397319356 1634798647 884655054 726584192 290446480 1542357501 2104272333 68364081 1029313865 450814187 1679237848 2114042808
95
1241258375 1966880262 351698344 455246 385894744 1123062722 724899226 1469152441 2025084031 2039022567 1037780472 2028834369 1182381586 788408519 1002838055 2006667892 1003194864 75313271 565832258 1662406719 35724868 1335177126 1191464316 326599737 330760836 967040598 1379967963 200604972 481512220 323074868 1157184157 1674673317 147816004 40459830 1166681689 1671641585 1924837209 429141899 1166093670 148529085 460097838 554305599 2035924233 1741443552 1343828389 498269629 991619310 1909038194 1145775694 806246893 1983996094 20960362 973953603 1834227837 1425851907 1301307906 1414600074 781295371 158567570 73856484 1016034582 1613179030 915870681 1448291210 1518513797 806311188 1234187617 1913748615 1508763498 2073138892 238443573 1666384449 724090608 805421409 1289514569 984120961 1398647346 1105404310 1994455364 570311209 363057283 1420437836 1656714723 1380312483 1584874494 59562811 1527305813 1268118362 235515588 1820400224 1368505605 1623928696 1304915618 1517516541 955604896
95
1420945666 881899063 190177901 204138264 295995600 1959427902 1061429005 2040715326 1205363570 246846161 1412922038 424162127 802958147 1764660153 2043791270 1354021549 886963863 1644860710 1748061755 441054416 234731146 796734660 334166378 158391167 1898847865 1333310753 262238557 863259913 664941253 2004716120 1780854777 1248405862 1021234556 1956916207 372380492 1953805680 2068898095 617926971 84825730 611336396 142570913 1016530493 188860320 633278907 1451097796 525734926 288093070 68798237 1113655341 1481703442 1774387672 1872673508 2035004761 1766871276 358142619 1446282383 1785544626 1856690889 1044083396 2025553088 1432813827 920132436 539905693 221861846 1780225135 432417239 46184051 368779342 74283516 1055289388 1422235985 1753122130 684554255 1766427615 249917528 384111809 632738771 1699642956 2045886653 2034821898 1072258341 982437514 723436005 570730001 1626330678 1651209034 649611151 516660182 1296016386 552459622 1336822895 577293921 1759516694 53971130 983843336
95
2114917204 1034380284 1209477184 1839426564 1343234247 1718199293 1975355581 692590088 65464872 819688575 775706291 895831233 415191030 2041987339 479115034 1952069240 1443064149 31873322 1703267034 1959295968 1846237099 1101625634 1413888441 1159752141 747141552 1554732671 1249224221 1739944716 1681462012 1656555546 2101065097 1466682701 1637809853 283581258 177631884 2005189964 33762121 1396451966 1678850725 1911534329 1627860178 1996789916 1005064422 924978935 1436809354 969250634 1600594337 1625944730 768190820 1514874458 1113155023 1679113904 1254616054 1198789771 922016982 1155166160 979255600 856703508 136985497 1862191017 1858216649 185799539 897362510 844833247 853611031 64986135 1337506952 1774142429 1692953604 59668676 1619940026 1569538112 1849824643 1664840342 1625574218 2016125218 795810444 841611315 955281520 306184613 580911509 1522181976 52296555 1614571084 382315332 1446922657 1821825823 1465293255 1946543200 422612157 1099836117 1369266295 498273648 2048788954 166598149
95
1899383772 396717739 1976830436 2118372485 1938412304 137033065 1750320255 666424006 1587593708 1727127147 2025519404 1713141047 1108828531 330785075 634974350 596275618 184408312 669845592 1015340026 1503308593 165176855 1875863101 899265398 966125932 1067839825 553328630 743486783 2123522085 1456075241 1236608536 1205598893 1220643749 1470699597 1213398027 1086755878 2053582336 392517019 1069894593 1107898296 1359304536 1117756719 2113852771 499097851 1139200024 1155025109 427542650 347850063 67672110 328050440 734876198 817039645 483614106 258820107 885323442 1089448332 1836724996 1372784981 1210687641 1925547434 1973130893 451490217 1589565667 1580020725 1618771557 735653040 1392290018 1209377347 860528062 1929135161 1954745833 842356090 566616885 507061864 190698017 1321895382 2049934811 1083797188 1823894078 2046714205 440752490 1811639175 236657821 1226072408 105097507 1758496758 947993514 1147171623 851566088 1732451889 613399970 567203839 675315871 484658057 1039937677 542192957
95
1206305046 683532063 1417417055 1986498183 265155620 1552522490 336401693 1164744798 140558958 175737928 734686348 195579170 1133120938 395884271 1524514921 184057194 202991060 2077904334 300156512 539012494 1058470383 359616845 894142385 756444287 1654636735 829782055 816108894 1658183390 588185925 1893960203 1970497097 325685311 177481883 728657939 1968862198 1648537080 1612547986 1549252163 1927187932 748829108 139758534 2141456532 1262655296 287450497 1055281809 346025928 2056781101 547952998 911558389 1281459633 264456208 396327627 1354727402 1106985371 1594409411 1342781456 568995420 520201865 843050566 465195261 98691189 1657694018 672913960 685495422 91617425 2128200721 274339736 317643744 584866575 11091631 934550341 306987898 1798729160 1142098640 774634442 1571010392 1939272355 621975668 1695830080 1919533175 456297811 327945746 242518465 179328075 1506774624 886431103 1134276195 1138669100 954887568 1414151018 1798258944 1264650322 1173061655 467051360 1430936960
95
1069427208 1082285639 1513999373 334636464 1792021283 1860468649 853389147 1190861218 31078753 196471459 1763438899 665420432 856534905 1393795739 981968515 2038937636 1884343706 162671801 854555836 2024442586 1752495843 34406289 295852639 270352892 1290122794 332879869 1469171413 1070223397 317105141 413821074 1859634039 1889292709 660831802 1785398608 1939621843 1517366181 2025196924 610851541 1418043823 1433588595 2140350377 1887393620 1938927743 196060010 1486452578 961778295 1449362008 373382391 1480190027 199980309 2040731877 886020083 503339501 633389717 1849408796 637983808 557725664 1211310436 606587397 332721346 1275818849 1676083777 533057710 356233173 1875601879 1464652777 820029897 149478830 1289311315 1486918236 1365884335 1967916418 1591434437 1755969661 1015214264 1939609034 784502491 166242346 55443826 359352254 1654986384 619212278 175853778 1068291292 670218757 1074881672 774999707 257537018 1476558071 1509265357 802972885 1320357697 939290090 1828627734 302723000
95
1428220692 296619954 1640339294 2030515945 1284441906 737613216 233111304 1551415773 540016862 674909789 646530094 1341269784 1115923568 1136672223 1753149607 1551872473 669417113 1729524099 406566352 1654427198 1613323918 1740184749 189675225 874111727 2029861155 559548533 1643245982 1867810785 280470437 774555576 2140702636 2094948915 1530438498 1257848716 959448079 674653463 1333839080 1311328804 1539742061 1542499629 753940591 1084349499 35347323 123579966 1363941851 708868963 936071037 1492753784 725175993 1027174730 2086856383 1857324040 1457171502 1673510573 241178298 1492924886 5540444 1924673950 1849713659 1833549919 330412589 469639416 754489436 2074521120 1076319276 866040496 1716292158 346851342 1681439098 706523081 1184938309 437343191 1282825785 336533585 1144824811 1459551697 1585495115 615924021 1108723979 1783286453 1679140039 309341771 1325123348 63885624 354842505 275292838 1920270008 1029686576 1029835740 1193212739 539648087 451548427 1998550045 1481656700 267603160
95
434791240 847353420 339037613 669703435 575343517 528554750 1347839051 1063386113 1847337163 2041896909 1240503399 725357458 836518783 1728400950 1563644865 1530469081 1873375591 2074457399 1025226698 223565397 191696364 603305456 1378271605 1733145924 1506076491 181610343 1506285236 1108104267 347649097 1039325825 1032414203 1782147398 656236000 1112353675 1765404107 1256395008 706509111 1575322742 2145971788 640634851 574506420 639831192 1108652872 404900014 646872701 444779450 1759824979 1394951499 135275136 281470874 61786946 1403735169 1598963760 1503825003 621767688 1736540208 1191585057 1002187648 943079008 250526066 553872547 2128936824 221937148 7713586 1475190709 1205647110 145275437 544522998 24644138 1923796441 1047180684 1484821779 1127116046 2059339073 497390960 653577132 1709932763 1864628209 889624906 655953050 430396623 866198553 1524743255 212576839 928369930 208220456 790171788 142911160 325989486 1421925333 1246520142 1240801651 1884806828 233681388 367846309
