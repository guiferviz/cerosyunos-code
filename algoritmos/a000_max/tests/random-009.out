2136293171
2072936011
2142885048
2136886033
2128691439
2136026421
2142279313
2145494462
2144726361
2133757847
2143709966
2121898685
2113450068
2110940202
2140331603
2141523207
2141460332
2074117296
2106580515
2138510445
2138833021
2105399970
2144009516
2073138892
2068898095
2114917204
2123522085
2141456532
2140350377
2140702636
2145971788
